### PR TITLE
OSDOCS-9316#Correct Azure File in-tree migration info

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-9316

Link to docs preview: https://70247--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-azure-file

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**: @gcharto @bertinatto

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
Same feedback as other similar PRs

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: storage/persistent_storage/persistent-storage-azure-file.adoc [hunk 1]
```diff
@@ -30,9 +30,9 @@ Azure File volumes use Server Message Block.
 
 [IMPORTANT]
 ====
-In future {product-title} versions, volumes provisioned using existing in-tree plugins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+{product-title} 4.13 and later provides automatic migration for the Azure File in-tree volume plugin to its equivalent CSI driver.
 
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
+CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
 ====
```
Comment (bertinatto) [inline storage/persistent_storage/persistent-storage-azure-file.adoc:36]: LGTM.

For reference: https://issues.redhat.com/browse/STOR-737

Comment (bertinatto) [reply storage/persistent_storage/persistent-storage-azure-file.adoc:36]: Actually, shouldn't this be targetting the 4.13 branch?

Comment (gcharot) [reply storage/persistent_storage/persistent-storage-azure-file.adoc:36]: LGTM and yes applies to 4.13+

Comment (lpettyjo) [reply storage/persistent_storage/persistent-storage-azure-file.adoc:36]: @bertinatto @gcharot this fix is targeted at 4.13+. OK?

Comment (gcharot) [reply storage/persistent_storage/persistent-storage-azure-file.adoc:36]: yes correct, thanks

**Pull request merged**